admin/print_page.html

```diff
@@ -15,7 +15,7 @@
   </head>
   <body>
     <div class="container">
-      <h1 id="fName"></h1>
```

Comment:
@Yinon-bar did we have to change this? Aren't we losing the ability to separate first from last?